common/kotlinx-coroutines-core-common/src/Dispatched.kt

```diff
@@ -21,26 +21,33 @@ internal object UndispatchedEventLoop {
     @JvmField
     internal val threadLocalEventLoop = CommonThreadLocal { EventLoop() }
 
-    inline fun execute(continuation: DispatchedContinuation<*>, contState: Any?, mode: Int, block: () -> Unit) {
+    inline fun execute(continuation: DispatchedContinuation<*>, contState: Any?, mode: Int, doYield: Boolean = false, block: () -> Unit) : Boolean {
```

Comment:
1. Boolean result needs documentation on its meaning
2. This line (with addition of doYield parameter) is too long.